src/jit/codegenlegacy.cpp

```diff
@@ -18830,15 +18830,17 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                                                    (ssize_t)stubAddr);
                             // The stub will write-back to this register, so don't track it
                             regTracker.rsTrackRegTrash(compiler->virtualStubParamInfo->GetReg());
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, REG_JUMP_THUNK_PARAM,
```

Comment:
@BruceForstall It sounds reasonable for me. I hope we'll switch to RyuJIT in short time.